### PR TITLE
refactor(locators): Locators have been replaced with more resilient ones to better handle layout changes.

### PR DESCRIPTION
**Pull request by chalapkoStanislav**
Locators have been replaced with more resilient ones to better handle layout changes.

Updated screenshots according to the new locators
Modified tests according to the new locators
Added fixes to improve test stability
Updated the README as some information was outdated.

### CODE REVIEW

**Review by tdelatorre — CHANGES_REQUESTED** [review]
Good job! I've put some comments. Two more suggestions:

- I've read the commit messages and in this case it's better to have a single commit for the whole PR.

- I've run the tests and some of them failed. Could you please check it?

File: pages/workspace/design-panel-page.js [hunk 1]
```diff
@@ -277,9 +237,9 @@ exports.DesignPanelPage = class DesignPanelPage extends BasePage {
     this.gridTypeField = page.locator(
       'div[class*="grid__option-row"] div[class*="type-select-wrapper"]',
     );
-    this.gridTypeSelectorSquareOption = page.locator('span:has-text("Square")');
-    this.gridTypeSelectorColumnsOption = page.locator('span:has-text("Columns")');
-    this.gridTypeSelectorRowsOption = page.locator('span:has-text("Rows")');
+    this.gridTypeSelectorSquareOption = page.getByText('Square');
+    this.gridTypeSelectorColumnsOption = page.getByText('Columns');
+    this.gridTypeSelectorRowsOption = page.getByText('Rows');
```
[the remark this answers is not in the document]
Comment (chalapkoStanislav) [reply pages/workspace/design-panel-page.js]: Added { exact: true } to the remaining locators.

**Comment by chalapkoStanislav** [comment]
[Ran the tests](https://github.com/penpot/penpotqa/actions/runs/11480574303), 7 failed because:

6 tests failed due to new accounts landing directly in the workspace instead of the dashboard (this changed today on PRE).
1448 - screenshot.

**Comment by tdelatorre** [comment]
> [Ran the tests](https://github.com/penpot/penpotqa/actions/runs/11480574303), 7 failed because:
> 
> 6 tests failed due to new accounts landing directly in the workspace instead of the dashboard (this changed today on PRE). 1448 - screenshot.

You're right! This is related to the [A/B test](https://tree.taiga.io/project/penpot/us/8138) that was enabled yesterday. Could you please run it again today? (A/B testing is now disabled)

**Comment by tdelatorre** [comment]
Changes reviewed! Just one task and it's approved: a single commit for the whole PR.

**Review by tdelatorre — APPROVED** [review]
Condition: you should have a single commit for the whole PR (because all commits are about the same thing)

**Pull request merged**